Main.java

```diff
@@ -9,7 +11,22 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // creates new scanner
+    Scanner input = new Scanner(System.in);
+    //asks user 
+    System.out.println("Please enter in two integers, on seperate lines, to divide.");
+    //first variable
+    int x = input.nextInt();
+    //second variable
+    int y = input.nextInt();
+    // qoucient
+    int q = x/y;
+    // calculates the remainder
+    int r = q % 2;
```

Comment:
Generally we want to avoid using acronyms or shorthand unless it is a well known one. I know what you are saying with it so it is not a huge deal, but just something to keep in mind for next time. It would have been better to called them quotient and remainder. Otherwise, great work!